deploy/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+namespace: open-cluster-management-hub
```

Comment:
Namespace is not created when running `make deploy-addon-manager`. Might needs a `namespace.yaml` under resources.
```
apiVersion: v1
kind: Namespace
metadata:
  name: open-cluster-management-hub
```